book/src/03_2_cargo_generate.md

```diff
@@ -13,6 +13,12 @@ $ cargo generate --git https://github.com/esp-rs/esp-idf-template cargo
 
 You'll be prompted for details regarding your new project. When given a choice between several options, navigate using cursor up/down and select with the Return key.
 
+The first message you see will be:
+`⚠️Unable to load config file: /home/$USER/.cargo/cargo-generate.toml`. You see this error because you do not have a favorite config file, but you don't need one and you can ignore this warning.
+
+🔎 You can create a [favorite config file](https://rustrepo.com/repo/cargo-generate-cargo-generate) who will be placed in `$CARGO_HOME/cargo-generate`, and override it with `-c, --config <config-file>`. 
```

Comment:
Maybe https://crates.io/crates/cargo-generate would be a better link? RustRepo seems to be third-party and have quite a few ads.

Reply:
Amended!